packages/mobile/src/identity/matchmaking.ts

```diff
@@ -0,0 +1,137 @@
+import { ContractKit } from '@celo/contractkit'
```

Comment:
I think a comment in this file, or maybe in the README just explaining the whole architecture here would be pretty useful. Or maybe there's a design doc? I just haven't been playing super close attention and feel like there is a lot going on here, but overall flow isn't super clear

Reply:
https://docs.google.com/document/d/1SvQ7qD_ms2S7OFxWJ3wy5x3vvYeJFtMx2yU5TevEAic/edit#